<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBranchEdit</class>
 <widget class="QDialog" name="CBranchEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dienststellen</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtID">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtName1"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name 1:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtName2"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtName3"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtAdress1"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="txtAdress2"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="txtAdress3"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="txtAdress4"/>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="txtAdress5"/>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="txtHours"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name 2:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Name 3:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Adresse 1:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Adresse 2:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Adresse 3:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Adresse 4:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Adresse 5:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Öffnungszeiten:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tbvBranches">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdNew">
       <property name="text">
        <string>Neu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdDelete">
       <property name="text">
        <string>Löschen</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdClose">
       <property name="text">
        <string>Schließen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName1</tabstop>
  <tabstop>txtName2</tabstop>
  <tabstop>txtName3</tabstop>
  <tabstop>txtAdress1</tabstop>
  <tabstop>txtAdress2</tabstop>
  <tabstop>txtAdress3</tabstop>
  <tabstop>txtAdress4</tabstop>
  <tabstop>txtAdress5</tabstop>
  <tabstop>txtHours</tabstop>
  <tabstop>tbvBranches</tabstop>
  <tabstop>cmdNew</tabstop>
  <tabstop>cmdDelete</tabstop>
  <tabstop>cmdClose</tabstop>
  <tabstop>txtID</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
